erpnext_com/www/docs/user/manual/en/regional/india/gst-3b-report.md

```diff
@@ -0,0 +1,37 @@
+###  GSTR3B Report in ERPNext
+
+To generate GSTR3B Report in ERPNext navigate to Accounting > Goods and Services Tax (GST India) > GSTR 3B Report or simply search for GSTR 3B Report in awesomebar.
+
+Click New to generate a new report or select an existing report to update it or download JSON.
```

Comment:
please mention steps to reach the document itself

Reply:
The link to reach the document is already mentioned Accounting > Goods and Services Tax (GST India) > GSTR 3B Report